angular.json

```diff
@@ -95,21 +95,23 @@
             "scripts": []
           },
           "configurations": {
+            "development": {
+              "optimization": false,
+              "aot": false,
+              "buildOptimizer": false,
+              "sourceMap": true,
+              "extractLicenses": false,
+              "namedChunks": true,
+              "vendorChunk": true
+            },
```

Comment:
For some reason this is appearing in the diff - but it is already present on target branch

Reply:
I think you would have to rebase this branch on release/v6.0.0 because the commits from the previous branch with build time improvements was squashed when it was merged into release/v6.0.0